android/beagle/src/main/java/br/com/zup/beagle/android/components/Text.kt

```diff
@@ -75,13 +75,13 @@ data class Text(
 
         text.textColor?.let {
             observeBindChanges(rootView, it) { value ->
-                this.setTextColor(value)
+                value?.let { valueLet ->this.setTextColor(valueLet) }
```

Comment:
the name of variable valueLet it is not a suggestive name don't answer anything, what do you think about rename to  color? 
```suggestion
                value?.let { color ->this.setTextColor(color) }
```
I think this check it is not necessary too, check if passing null the behavior it is the same.